frontend/src/index.tsx

```diff
@@ -54,4 +57,9 @@ if (!window.location.pathname.includes(rootBasename)) {
   router.navigate(`/${window.location.search}`, { replace: true });
 }
 
+// Suppress thousands of console warnings when scrolling over PFT graphs
+configure({
+  enforceActions: 'never'
+});
```

Comment:
Should we be suppressing these or are the warnings indicative of a problem with our implementation?

Reply:
I think the issue is rooted in PFT.  We get thousands of warnings as we move the cursor over the graph.  It seems to be a PFT thing, not something we are doing.

Reply:
I can open an issue with PFT. I've noticed these warnings too and wondered about them but didn't do anything about it.

Reply:
Before opening an issue, I guess I should ensure it's not somehow our problem.  I think we can leave it suppressed while we look into it.  I'm slight;y suspicious now, because when mousing around in their "surge" demo, I don't see the warnings.

Reply:
I only see the warnings in dev mode I don't see them with the production build of the UI.

Reply:
Well, now I'm not sure.  This happens when we call `elem.setData({some new set of data})`.  But I'm not sure why that's a problem.  the message says `Since strict-mode is enabled, changing (observed) observable values without using an action is not allowed`.  So, maybe it's an issue in PFT's impl of `setData()`, because under the covers the `data` field is tied to a mobX Observable.  Maybe I'll ask about this in PFT Slack...

Reply:
OK, well that was a pita, but I believe we've gotten rid of the warnings and now perform element updates in the way PFT wants us to do it.  This may give us a small boost in responsiveness as the user mouses across the graph, not sure, but now we batch together a bunch of highlighting updates.

code is pushed.